zstacklib/zstacklib/utils/shell.py

```
@@ -2,20 +2,28 @@
+
 # instead of closing [3, ulimit -n)
 def _linux_close_fds(self, but):
-    for fd in [ int(n) for n in os.listdir('/proc/%d/fd' % os.getpid()) ]:
+    for fd in [int(n) for n in os.listdir('/proc/%d/fd' % os.getpid())]:
         if fd > 2 and fd != but:
-            try: os.close(fd)
-            except: pass
+            try:
+                os.close(fd)
+            except:
+                pass
```

Ruff (0.12.2)

Use `contextlib.suppress(BaseException)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(BaseException)`

(SIM105)